pretix_eth/models.py

```diff
@@ -1,70 +1,43 @@
-from django.db import (
-    models,
-    transaction,
-)
-from pretix.base.models import (
-    Event,
-    OrderPayment,
-    OrderRefund,
-)
+from django.db import models
+from django.utils import timezone
 
+from pretix.base.models import OrderPayment
 
-class WalletAddressError(Exception):
-    pass
 
+class SignedMessage(models.Model):
 
-class WalletAddressQuerySet(models.QuerySet):
-    def for_event(self, event: Event) -> models.QuerySet:
-        return self.filter(event=event)
+    MAX_AGE_UNCONFIRMED = 30 * 60  # allow retry payment after 30mins
```

Comment:
we have replicated this at many places. Let's move it to a config.yaml or something?

Reply:
This was class constant actually not used in the end, and it's been moved to settings.